cncf_process/incubation_proposal/incubation_proposal.md

```diff
@@ -0,0 +1,179 @@
+  What's more, we introduced `karmada-descheduler` to reschedule in case of cluster resource imbalance. Based on this, users can distribute their applications more easily among clusters across regions and vendors.
+* **Easier to manage the cluster and the resources on the cluster**. In the past year, we introduced a global uniform resource view across clusters.
+  - We enhanced the framework of the resource interpreter to enable users to customize the resource status collected to the control plane.
+  - We introduced the aggregated API to allow users to connect and operate member clusters from the control plane.
+  - We introduced a new component `karmada-search` to implement global search and global proxy for multi-cluster resources.
+    Based on this, users no longer have to deal with tedious and repetitive configuration management, nor to endure the management costs caused by fragmented API access portals.
+* **More convenient cross-cluster service governance**. We support multi-cluster service discovery and a one-stop multi-cluster network solution. Now users can enable service governance for cross-cluster with Karmada.
+* **High availability for multi-cloud applications**. We support cross-cluster failover of applications.
+  When a cluster fails, applications can smoothly migrate from the failed cluster to another available cluster. During the replicas migration process, Karmada ensures that the service replicas do not drop to zero, thereby ensuring that the service will not be interrupted.
+  Based on this, users can use Karmada to manage distributed applications with confidence.
+* **Smoother multi-cluster O&M experience**. In the past year, We continue to optimize our command line tools. Now users can complete the creation of the Karmada control plane and bootstrap token-based cluster registration in just a few clicks.
+  Meanwhile, we provide not only a smooth version upgrade path but also a clear approach to migrate from Kubefed or promote legacy workloads from Kubernetes.
+
+## Incubation Stage Requirements
+
+**_Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope._**
+
+Now Karmada now has 11 adopters that using Karmada in `production` environments as of Oct 25, 2022. The full adopter list can be found in our [website](https://karmada.io/docs/next/casestudies/adopters).
+
+Highlighted examples include:
+
+* **[VIPKID](https://www.vipkid.com/)**: Use Karmada to build a multi-cloud PaaS platform with multi-vendor and multi-region. The use case details can be found [here](https://karmada.io/docs/next/casestudies/vipkid).
+* **[ICBC](http://www.icbc.com.cn/icbc/)**: Use Karmada to build a multi-cloud platform to manage **100** Kubernetes clusters and more than **280,000** containers. The use case details can be found [here](https://karmada.io/zh/blog/2022/03/22/multi-cluster-management-with-an-ocean-of-nodes/).
```

Comment:
```suggestion
* **[ICBC](http://www.icbc.com.cn/icbc/)**: Uses Karmada to build a multi-cloud platform to manage **100** Kubernetes clusters and more than **280,000** containers. The use case details can be found [here](https://karmada.io/zh/blog/2022/03/22/multi-cluster-management-with-an-ocean-of-nodes/).
```

Reply:
Replace the links with the English ones for all customer cases.

Reply:
AIML INSTITUTE only has Chinese website.